poc-cb-net/cmd/admin-web/admin-web.go

```diff
@@ -173,24 +173,24 @@ func WebsocketHandler(c echo.Context) error {
 	}
 }
 
-func configurationInformationHandler(etcdClient *clientv3.Client, responseText []byte) {
+func cladnetSpecificationHandler(etcdClient *clientv3.Client, responseText []byte) {
```

Comment:
```suggestion
func CLADNetSpecificationHandler(etcdClient *clientv3.Client, responseText []byte) {
```
?

Reply:
@seokho-son, thank you for your review comment ☺️ 

My intention was to use the function internally. I named the function with lowercase letters 😢

Reply:
ㅠㅠ